tests_e2e/tests/lib/virtual_machine_client.py

```diff
@@ -54,9 +57,9 @@ def __init__(self, vm: VmIdentifier):
             base_url=cloud.endpoints.resource_manager,
             credential_scopes=[cloud.endpoints.resource_manager + "/.default"])
 
-    def get_description(self) -> VirtualMachine:
+    def get_model(self) -> VirtualMachine:
```

Comment:
I call this method "get_description" because I wasn't sure what is returning and the documentation just said that it returns the "description" of the VM. It actually returns the model.